src/cmd/serial.js

```diff
@@ -897,7 +898,7 @@ module.exports = class SerialCommand {
 			}
 
 			function startTimeout(to){
-				self._serialTimeout = setTimeout(() => reject('Serial timed out'), to);
+				self._serialTimeout = setTimeout(() => reject(timeoutError), to);
```

Comment:
one of the many clean-up tasks we have is to start using real errors when rejecting